File: src/main/java/top/zbeboy/isy/web/vo/graduate/design/replan/GraduationDesignReplanAddVo.java
package top.zbeboy.isy.web.vo.graduate.design.replan;

import lombok.Data;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

/**
 * Created by lenovo on 2017-07-09.
 */
@Data
public class GraduationDesignReplanAddVo {
    @NotNull
    @Size(max = 64)
    private String graduationDesignReleaseId;
    @NotNull
    private String paperTime;
    @NotNull
    private String defenseTime;
    private String[] dayDefenseStartTime;
    private String[] dayDefenseEndTime;
    @NotNull
    private int intervalTime;
    private String defenseNote;
}

File: src/main/java/top/zbeboy/isy/web/vo/graduate/design/replan/GraduationDesignReplanAddVo.kt
package top.zbeboy.isy.web.vo.graduate.design.replan

import javax.validation.constraints.NotNull
import javax.validation.constraints.Size

/**
 * Created by zbeboy 2018-02-06 .
 **/
open class GraduationDesignReplanAddVo {
    @NotNull
    @Size(max = 64)
    var graduationDesignReleaseId: String? = null
    @NotNull
    var paperTime: String? = null
    @NotNull
    var defenseTime: String? = null
    var dayDefenseStartTime: Array<String>? = null
    var dayDefenseEndTime: Array<String>? = null
    @NotNull
    var intervalTime: Int = 0
    var defenseNote: String? = null
}